main.tf

```diff
@@ -131,6 +131,7 @@ locals {
       bucket_name                       = local.bucket_name
       shared_cache                      = var.cache_shared
       sentry_dsn                        = var.sentry_dsn
+      listen_address                    = var.listen_address
```

Comment:
question: could this name conflict with the session_server.listen_address setting. If so I suggest to rename it.

Reply:
i renamed it just in case :)